mantidproject/mantid
# Remove scons-local from installers/WinInstall directory

Fixes #15417 

The scons-local directory has been removed as it is no longer required due tot he changes made in #15206
# To Test
- Ensure you are able to create an windows installer using cpack
- Ensure that the `installers/WinInstaller/scons-local` no longer exists
- Ensure that all tests pass

_No release notes required for this change as it does not affect users_
